content/SCALE/SCALEUIReference/Shares/SMBSharesScreens.md

```diff
@@ -160,7 +160,7 @@ The **Other Options** settings are for improving Apple software compatibility, Z
 | **Time Machine** | Enables [Apple Time Machine](https://support.apple.com/en-us/HT201250) backups on this share. This option requires SMB2/3 protocol extension support. You can enable this in the general SMB server configuration. |
 | **Legacy AFP Compatibility** | Select to enable the share to behave like the deprecated Apple Filing Protocol (AFP). Leave cleared for the share to behave like a normal SMB share. This option controls how the SMB share reads and writes data. Only enable this when this share originated as an AFP sharing configuration. You do not need legacy compatibility for pure SMB shares or macOS SMB clients. This option requires SMB2/3 protocol extension support. You can enable this in the general SMB server configuration. |
 | **Enable Shadow Copies** | Select to export ZFS snapshots as [Shadow Copies](https://docs.microsoft.com/en-us/windows/win32/vss/shadow-copies-and-shadow-copy-sets) for Microsoft Volume Shadow Copy Service (VSS) clients. |
-| **Export Recycle Bin** | Select to enable. Deleted files from the same dataset move to the **Recycle Bin** and do not take any additional space. Deleting files over NFS removes the files permanently. Files in a different dataset or a child dataset copy to the dataset with the recycle bin. To prevent excessive space usage, TrueNAS deletes files larger than 20 MiB rather than moving them. Adjust the **Auxiliary Parameter** by adding the `crossrename:sizelimit=` setting to allow larger files. For example, `crossrename:sizelimit=<i>50</i>` allows moves of files up to 50 MiB in size. Export Recycle Bin permanently deletes or moves files from the recycle bin. It is not a replacement for ZFS snapshots. |
+| **Export Recycle Bin** | Select to enable. Deleted files from the same dataset move to a **Recycle Bin** in that dataset and do not take any additional space. Deleting files over NFS removes the files permanently. |
```

Comment:
Wondering if it wouldn't be better to use the same language in both the CORE and SCALE page. Specifically the CORE update removes the statement that deleting files over NFS is permanent and this one retains it. That seems like it should be consistent.